shared/rest/src/main/java/io/pravega/shared/rest/RESTServer.java

```diff
@@ -71,7 +71,7 @@ protected void startUp() {
                 contextConfigurator.setKeyStoreFile(restServerConfig.getKeyFilePath());
                 contextConfigurator.setKeyStorePass(JKSHelper.loadPasswordFrom(restServerConfig.getKeyFilePasswordPath()));
                 httpServer = GrizzlyHttpServerFactory.createHttpServer(baseUri, resourceConfig, true,
-                        new SSLEngineConfigurator(contextConfigurator, false, false, false));
+                        new SSLEngineConfigurator(contextConfigurator, false, false, false).setEnabledProtocols(restServerConfig.tlsProtocolVersion().split(",")));
```

Comment:
Same comment as the GRPCServiceConfig.

Reply:
Moved away the split or regex logic from here.